cEP-0024.md

```diff
@@ -0,0 +1,252 @@
+# Implement GitMate automations and plugins for coala
+
+| Metadata |                                                       |
+| -------- | ----------------------------------------------------- |
+| cEP      | 24                                                    |
+| Version  | 1.0                                                   |
+| Title    | Implement Gitmate automations and plugins for coala   |
+| Authors  | Vamshi Krishna Bommerla <mailto:bommvams72@gmail.com> |
+| Status   | Proposed                                              |
+| Type     | Process                                               |
+
+## Abstract
+
+This cEP describes the improvements to the coala workflow and the necessary
+automation in [GitMate](https://gitmate.io/) to realise it, as a part of the
+[GSoC project](https://summerofcode.withgoogle.com/projects/#5344316227059712).
+
+## Introduction
+
+[GitMate](https://gitmate.io/) is an automation tool for developers. While it
+works well, there are a number of missing features which would save time of
+coala community.
+
+Many of the plugins in GitMate are general plugins, but coala needs coala
```

Comment:
frame it like this: 

> but we need coala specific plugins to support the development workflow.

Reply:
No, I can't use `we` in cEP.

Reply:
ok. Then just do what it was before I commented on it. Doesn't look good with ``coala`` being repeated so many times.